src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`3.cs

```diff
@@ -136,33 +136,29 @@ protected override void SetupNewTextView(IVsTextView textView)
                 {
                     // We need to get our outlining tag source to notify it to start blocking
                     var outliningTaggerProvider = this.Package.ComponentModel.GetService<OutliningTaggerProvider>();
-                    ProducerPopulatedTagSource<IOutliningRegionTag> tagSource;
-                    if (outliningTaggerProvider.TryRetrieveTagSource(wpfTextView, wpfTextView.TextBuffer, out tagSource))
```

Comment:
No need for anyone outside of the tagger infrastructure to know about tag sources now.